code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -377,7 +377,7 @@ GLOBAL_LIST_INIT(adamantine_recipes, list(
  */
 
 /obj/item/stack/sheet/mineral/runite
-	name = "Runite"
+	name = "runite"
```

Comment:
wtf are you doing

Reply:
i was editing this file already so i wanted consistency with all other materials